openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -709,604 +635,510 @@ const TeamDetailsV1 = ({
-              {heading}
-            </Typography.Title>
-            {isActionAllowed() && (
-              <Tooltip
-                placement="right"
-                title={
-                  entityPermissions.EditAll || entityPermissions.EditDisplayName
-                    ? t('label.edit-entity', {
-                        entity: t('label.display-name'),
-                      })
-                    : t('message.no-permission-for-action')
-                }>
-                <Button
-                  className="p-0"
-                  data-testid="edit-synonyms"
-                  disabled={
-                    !(
-                      entityPermissions.EditDisplayName ||
-                      entityPermissions.EditAll
-                    )
-                  }
-                  icon={<IconEdit height={16} width={16} />}
-                  size="small"
-                  type="text"
-                  onClick={() => setIsHeadingEditing(true)}
+          </Col>
+          <Col>
+            <Space align="center">
+              <span>
+                <Switch
+                  checked={showDeletedTeam}
+                  data-testid="show-deleted"
+                  onClick={onShowDeletedTeamChange}
                 />
-              </Tooltip>
-            )}
-          </Space>
-        )}
-      </div>
-    );
-  };
-
-  const teamTypeElement = useMemo(() => {
-    if (currentTeam.teamType === TeamType.Organization) {
-      return null;
-    }
+                <Typography.Text className="m-l-xs">
+                  {t('label.deleted')}
+                </Typography.Text>
+              </span>
 
-    return (
-      <>
-        {t('label.type') + ' - '}
-        {currentTeam.teamType ? (
-          showTypeSelector ? (
-            <TeamTypeSelect
-              handleShowTypeSelector={setShowTypeSelector}
-              parentTeamType={
-                last(parentTeams)?.teamType ?? TeamType.Organization
-              }
-              showGroupOption={!childTeams.length}
-              teamType={currentTeam.teamType ?? TeamType.Department}
-              updateTeamType={
-                entityPermissions.EditAll ? updateTeamType : undefined
-              }
-            />
-          ) : (
-            <>
-              {currentTeam.teamType}
-              {entityPermissions.EditAll && (
-                <Icon
-                  className={classNames('vertical-middle m-l-xs', {
-                    'opacity-50': isGroupType,
-                  })}
-                  data-testid="edit-team-type-icon"
+              {createTeamPermission && (
+                <Button
+                  data-testid="add-team"
                   title={
-                    isGroupType
-                      ? t('message.group-team-type-change-message')
-                      : t('label.edit-entity', {
-                          entity: t('label.team-type'),
-                        })
+                    createTeamPermission
```

Comment:
We won't render button if there's no permission
We can avoid unwanted condition here